hadoop-ozone/dist/src/main/compose/ozone/test.sh

```diff
@@ -35,14 +35,14 @@ execute_robot_test scm gdpr
 
 for scheme in ofs o3fs; do
   for bucket in link bucket; do
-    execute_robot_test scm -v SCHEME:${scheme} -v BUCKET_TYPE:${bucket} ozonefs/ozonefs.robot
+    execute_robot_test scm -v SCHEME:${scheme} -v BUCKET_TYPE:${bucket} -N ozonefs-${scheme}-${bucket} ozonefs/ozonefs.robot
```

Comment:
Nice.  Can you also add explicit name for the different `readdata` tests in `ozone-topology`?

https://github.com/apache/hadoop-ozone/blob/ddd3211757ad67d1574e0e9e983b182681050a7c/hadoop-ozone/dist/src/main/compose/ozone-topology/test.sh#L42

https://github.com/apache/hadoop-ozone/blob/ddd3211757ad67d1574e0e9e983b182681050a7c/hadoop-ozone/dist/src/main/compose/ozone-topology/test.sh#L52

Reply:
:+1: